packages/components/src/components/@shared/alert-fc/component.tsx

```diff
@@ -0,0 +1,68 @@
+import clsx from 'clsx';
+import type { FunctionalComponent as FC } from '@stencil/core';
+import { h } from '@stencil/core';
+import { Log } from '../../../schema';
+import type { JSXBase } from '@stencil/core/internal';
+import { AlertIcon, CloserButton, Content, HeadingContent } from './components';
+import { DEFAULT_PROPS, type KolAlertFcProps } from './_types';
+
+const KolAlertFc: FC<KolAlertFcProps> = (props, children) => {
+	const {
+		class: classNames = {},
+		_type = DEFAULT_PROPS._type,
+		_variant = DEFAULT_PROPS._variant,
+		_label,
+		_hasCloser,
+		_alert,
+		onVibrationTimeout,
+		onCloserClick,
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		_on,
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		_level,
+		...other
+	} = props;
+
+	if (_alert && navigator.vibrate) {
+		/**
+		 * - https://developer.mozilla.org/de/docs/Web/API/Navigator/vibrate
+		 * - https://googlechrome.github.io/samples/vibration/
+		 */
+		try {
+			Log.debug(['Navigator should vibrate ...', navigator.vibrate([100, 75, 100, 75, 100])]);
```

Comment:
Ist es "sicher", in einer FC Side-Effects with `navigator.vibrate` zu nutzen, oder könnten sie unter Umständen ungewollt mehrfach ausgeführt werden?

Reply:
es kann ungewollt mehrfach ausgelöst werden.

Reply:
aber es wird auch jetzt schon mehrfach gerendert